SpecFlowProject1/pages/AddEditandDeleteSkills.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium;
+using SpecFlowProject1.utilities;
+using static System.Collections.Specialized.BitVector32;
+
+namespace SpecFlowProject1.pages
+{
+    public class AddEditandDeleteSkills
+    {
+        public void navigateToSkill(IWebDriver webdriver)
+        {
+            IWebElement navigate = webdriver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[1]/a[2]"));
```

Comment:
5/
Please implement the POM Pattern, please see the below 
https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html